utils/babel_runner.py

```diff
@@ -233,16 +233,13 @@ def _get_logger():
 
     os.chdir(ROOT)
     if action == "extract":
-        raise SystemExit(run_extract())
+        run_extract()
```

Comment:
I kind of like the raise SystemExit(func()). And I prefer having the correct exit code every time.

Reply:
the trouble is, the `func()` doesn't return anything. Did they return error codes at some point in the past?

Reply:
Oh. I didn't know. I don't think so then. At one time, I had asked Adam why using SystemExit instead of sys.exit() and I think they told me it's better that way. Now, if nothing is being returned..., then let's leave it like that.

Can you, however, change the if-if-if into if-elif-elif (it makes sure we don't have an if-duplicate, though it wouldn't happen, but better safe than sorry).

Reply:
have a look and see if you're happy with the approach

Reply:
I am very happy!